app/views/api/v1/projects/_activity.xml.erb

```diff
@@ -1,4 +1,4 @@
-<iati-activity xml:lang="en" last-updated-datetime="<%= project.updated_at.xmlschema %>" linked-data-uri="https://www.ngoaidmap.org/projects/<%= project.id %>" >
+<iati-activity xml:lang="en" last-updated-datetime="<%= project.updated_at.xmlschema %>" humanitarian="<% if project.sectors.any? { |s| s.id == 18 } %>1<% else %>0<% end %>" linked-data-uri="https://www.ngoaidmap.org/projects/<%= project.id %>" >
```

Comment:
I refactor this `<% if project.sectors.any? { |s| s.id == 18 } %>1<% else %>0<% end %>` into  a helper method.